Apps/AdminWebClient/src/Server/Controllers/DashboardController.cs

```diff
@@ -61,16 +61,16 @@ public IActionResult GetRegisteredUserCount()
         /// <summary>
         /// Retrieves the count of logged in user in the last day.
         /// </summary>
-        /// <param name="offset">The offset from the client browser to UTC.</param>
+        /// <param name="timeOffset">The offset from the client browser to UTC.</param>
         /// <returns>The count of logged in users in the current day.</returns>
         /// <response code="200">Returns the list of user feedbacks.</response>
         /// <response code="401">The client must authenticate itself to get the requested response.</response>
         /// <response code="403">The client does not have access rights to the content; that is, it is unauthorized, so the server is refusing to give the requested resource. Unlike 401, the client's identity is known to the server.</response>
         [HttpGet]
         [Route("LoggedInCount")]
-        public IActionResult GetTodayLoggedinUsersCount(int offset)
+        public IActionResult GetTodayLoggedinUsersCount(int timeOffset)
```

Comment:
@FuriousLlama I believe we need to add the date range to this one per req: "update service and delegate to accept date ranges " on 9772 right?

Reply:
Instead of performing the same query multiple times with different dates, it would be better if the front end pics what date it wants to show

Reply:
not sure if I understand the solution design for this one, as it doesn't seem to me this fulfills "in admin site, add dashboard stat for daily logins with date range selector (two dates)" requirement on the user story 9766. We may need to chat on our groom to make sure :-)